x-pack/plugins/siem/public/management/pages/policy/store/policy_list/mock_policy_result_list.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { GetPolicyListResponse } from '../../types';
+import { EndpointDocGenerator } from '../../../../../../common/endpoint/generate_data';
+
+export const mockPolicyResultList: (options?: {
+  total?: number;
+  request_page_size?: number;
+  request_page_index?: number;
+}) => GetPolicyListResponse = (options = {}) => {
+  const {
+    total = 1,
+    request_page_size: requestPageSize = 10,
+    request_page_index: requestPageIndex = 0,
+  } = options;
+
+  // Skip any that are before the page we're on
+  const numberToSkip = requestPageSize * requestPageIndex;
+
+  // total - numberToSkip is the count of non-skipped ones, but return no more than a pageSize, and no less than 0
+  const actualCountToReturn = Math.max(Math.min(total - numberToSkip, requestPageSize), 0);
```

Comment:
:red_circle:  Did you mean for that `0` to be `1`? Otherwise it seems like actualCountToReturn *could* be 0, depending on the values of total, numberToSkip, and RequestPageSize?

Reply:
That's intentional if we request a page that would have no results, for instance we request a page 3 where there are only 2 pages.  Imagine we're sitting on a page 3 in the UI.  20 min later, the user refreshes the page (with that same page index in the URL) and the records on that page have all been deleted, we would show page 3 again, but with 0 entries as opposed to sending an error.

Reply:
You should update the comment that says `and no less than 0` in that case